src/Entity/User.php

```diff
@@ -53,7 +54,7 @@ class User extends PublicKeyCredentialUserEntity implements UserInterface
         joinColumns:[new JoinColumn(name: "user_id", referencedColumnName: "id")],
         inverseJoinColumns:[new JoinColumn(name:"user_handle", referencedColumnName: "id", unique: true)]
     )]
-    protected ArrayCollectionAlias $publicKeyCredentialSources;
+    protected Collection $publicKeyCredentialSources;
```

Comment:
I'm OK with this change, but was it really needed to get you up and running? If not, consider removing it, keeping the PR more focused.